client/package.json

```diff
@@ -4,6 +4,9 @@
   "private": true,
   "proxy": "http://localhost:3000",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
```

Comment:
No need to add this dependency. Kindly use react icons only. That dependency is already included in the project.